Update instructions on how to contribute (#50)
PlasmaPy/plasmapy.github.io: README.md

<div align="center"><img src="https://raw.githubusercontent.com/PlasmaPy/PlasmaPy-logo/master/exports/with-text-dark.png" width="600"/></div>

# PlasmaPy Website

[![Build Status](https://travis-ci.org/PlasmaPy/plasmapy.github.io.svg?branch=src)](https://travis-ci.org/PlasmaPy/plasmapy.github.io)
[![Matrix](https://matrix.to/img/matrix-badge.svg)](https://riot.im/app/#/room/#plasmapy:openastronomy.org)
[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/PlasmaPy/Lobby)

> This repository contains the code for the website for the PlasmaPy
> project.  The URL of the website is: 

Reviewer: ```suggestion
This repository contains the code for the PlasmaPy website ([`https://www.plasmapy.org/`](https://www.plasmapy.org/)).
```

> [`https://www.plasmapy.org/`](https://www.plasmapy.org/)

Reviewer: ```suggestion
```

## Contributing

> The [contribution guide](./CONTRIBUTING.md) contains instructions on how to contribute to the PlasmaPy website, which includes aspects like how to fork and clone the repository, how to build local copies of the website, etc."

Reviewer: ```suggestion
The [contribution guide](./CONTRIBUTING.md) contains instructions on how to contribute to the PlasmaPy website, which includes aspects like how to fork and clone the repository, how to build local copies of the website, etc.
```

> <div align="center"><img src="https://raw.githubusercontent.com/PlasmaPy/PlasmaPy-logo/master/exports/with-text-dark.png" width="600"/></div

Reviewer 2: ```suggestion
<div align="center"><img src="https://raw.githubusercontent.com/PlasmaPy/PlasmaPy-logo/master/exports/with-text-dark.png" width="600"/></div>
```

# PlasmaPy Website

[![Build Status](https://travis-ci.org/PlasmaPy/plasmapy.github.io.svg?branch=src)](https://travis-ci.org/PlasmaPy/plasmapy.github.io)
[![Matrix](https://matrix.to/img/matrix-badge.svg)](https://riot.im/app/#/room/#plasmapy:openastronomy.org)
[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/PlasmaPy/Lobby)

This repository contains the code for the website for the PlasmaPy
project.  The URL of the website is: 
[`https://www.plasmapy.org/`](https://www.plasmapy.org/)

## Contributing

> The [contribution guide](./CONTRIBUTING.md) contains instructions
> on how to contribute code or content to PlasmaPy's website.

Reviewer 2: ```suggestion
The [contribution guide](./CONTRIBUTING.md) contains instructions on how to contribute to the PlasmaPy website, which includes aspects like how to fork and clone the repository, how to build local copies of the website, etc."
```
